src/chrome/content/rules/Zillow.com.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	Non-functional hosts
+		Timeout was reached:
+		- engineering.zillow.com
+
+		Certificate mismatched:
+		- investors.zillow.com
+
+		Status code mismatch:
+		- mortgageapi.zillow.com
+
+		Mixed content blocking (MCB) triggered:
+     		- info.zillow.com
+-->
+<ruleset name="Zillow.com">
+	<target host="zillow.com" />
+	<target host="www.zillow.com" />
+	<target host="premieragent.zillow.com" />
+	<target host="photos1.zillow.com" />
```

Comment:
Could you add a test url please? E.g. http://photos1.zillow.com/i_g/ISatx8mbaxt0v7.jpg